politeiad/backendv2/tstorebe/testing.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2020-2021 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package tstorebe
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sync"
+	"testing"
+
+	"github.com/decred/politeia/politeiad/backendv2/tstorebe/tstore"
+)
+
```

Comment:
This could stand some more tests.

Reply:
There used to be a lot more test coverage but I had to rip them out because things changed so much over the course of developing this PR.  I can probably start adding test coverage back in now that the changes have calmed down.

Reply:
Let's write an issue for this and do it post-release. There is too much code here to not run UT.